### PR TITLE
fix(K8s): Updated the Explore your Kubernetes data category

### PR DESCRIPTION
**Pull request by nbaenam**
Updated these docs:

- [Explore your Kubernetes cluster](https://docs.newrelic.com/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer/)
- [Find and use your Kubernetes data](https://docs.newrelic.com/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/find-use-your-kubernetes-data/)
- [Kubernetes events](https://docs.newrelic.com/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-events-integration/)
- [Plugin for log forwarding](https://docs.newrelic.com/docs/logs/forward-logs/kubernetes-plugin-log-forwarding/)

Ticket: NR-266821

### CODE REVIEW

File: src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx [hunk 1]
```diff
@@ -1,121 +1,128 @@
...
+3. Generate some traffic and wait a few minutes, then [check your account](#find-data) for data.
 
-You can install it in your cluster using our Helm chart in two ways.
+<InstallFeedback />
 
-### Use our guided install [#helm-guided-install]
+## Fluent Bit output plugin [#fluent-bit]
 
-Although the [newrelic-logging](https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-logging) chart works as a standalone, we recommend installing it as part of the [nri-bundle](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle) chart.
+New Relic has a [Fluent Bit](https://fluentbit.io/) [output plugin](https://github.com/newrelic/newrelic-fluent-bit-output) to forward your logs to New Relic log management. This plugin is also provided in a standalone Docker image that can be installed in a Kubernetes cluster in the form of a DaemonSet, also known as the Kubernetes plugin.
```
Comment (rhetoric101) [inline src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx:56]: @nbaenam I thought that the `DaemonSet` from New Relic was the Kubernetes plugin, which seemed to be a subset of the overall Kubernetes integration. Is that true? If so, are we just saying you could have two DaemonSets: one for monitoring pods (the Kubernetes plugin) and one for forwarding logs (the Fluent Bit image)? I was confused by the "...also known as the Kubernetes plugin."

File: src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx [hunk 2]
```diff
@@ -1,121 +1,128 @@
...
+    We capture Fluent Bit's internal metrics by using its [`prometheus_scrape` input plugin](https://docs.fluentbit.io/manual/pipeline/inputs/prometheus-scrape-metrics) in conjunction with its [`prometheus_remote_write` output plugin](https://docs.fluentbit.io/manual/pipeline/outputs/prometheus-remote-write). All the Prometheus `counter` metrics are actually cumulative counters, but we automatically perform a delta conversion when they are ingested at New Relic to ease querying them using NRQL later. You can find more details [here](/docs/data-apis/understand-data/metric-data/cumulative-metrics/).
 
-```bash
-    helm repo update newrelic
-```
+    * **[`newrelic-fluent-bit-output`'s internal plugin metrics](https://github.com/newrelic/newrelic-fluent-bit-output?tab=readme-ov-file#troubleshooting-metrics)**: Collected by the output plugin and sent to New Relic's metric API. These metrics only contain the `cluster_name` dimension, so they can be narrowed down to a particular cluster but not to a particular host or pod. They're useful to assess the overall latency when delivering the logs to the New Relic Logs API or to observe potential packaging problems.
```
Comment (rhetoric101) [inline src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx:68]:
```suggestion
    * **[Internal plugin metrics from `newrelic-fluent-bit-output`](https://github.com/newrelic/newrelic-fluent-bit-output?tab=readme-ov-file#troubleshooting-metrics)**: Collected by the output plugin and sent to New Relic's metric API. These metrics only contain the `cluster_name` dimension, so they can be narrowed down to a particular cluster but not to a particular host or pod. They're useful to assess the overall latency when delivering the logs to the New Relic Logs API or to observe potential packaging problems.
```

**Review by rhetoric101 — COMMENTED** [review]
@nbaenam thanks for making all of these improvements! I had a few suggestions and questions. Take a look.

File: src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/get-logs-version.mdx [hunk 3]
```diff
@@ -26,65 +26,61 @@ If you're using version 2 of the integration, see [Kubernetes logs in version 2]
...
+        # newrelic-infrastructure:
+        #   verboseLog: true
+
+        # To enable verboseLog for all sub-charts
+        # global:
+        #   verboseLog: true
         ```
-
+       
+        Once you have added the desired configuration, upgrade the solution with the following command:
```
[the remark this answers is not in the document]
Comment (nbaenam) [reply src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/get-logs-version.mdx:41]: Good catch, @rhetoric101, thank you!

File: src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/get-logs-version.mdx [hunk 4]
```diff
@@ -26,65 +26,61 @@ If you're using version 2 of the integration, see [Kubernetes logs in version 2]
...
         ```
 
-      Once you have edited the manifest, upgrade the solution with the following command:
+        After collecting your verbose logs, revert the change in your `values.yaml` file, and run the `upgrade` command again. 
+
+     - To get verbose logging details using a manifest file, you need to set two different environment variables:
+       1. `NRIA_VERBOSE="1"` for all `agent` and `forwarder` containers; this enables agent verbose logs. You can set this below `NRIA_LICENSE_KEY`.
+       2. set `NRI_KUBERNETES_VERBOSE=true"` in all components  of the integration; this enables verbose logs for the integration.
+
+        ```yaml
```
[the remark this answers is not in the document]
Comment (nbaenam) [reply src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/get-logs-version.mdx:56]: Fixed! Thanks!

File: src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/get-logs-version.mdx [hunk 5]
```diff
@@ -26,65 +26,61 @@ If you're using version 2 of the integration, see [Kubernetes logs in version 2]
...
+            key: licenseKey
+            name: newrelic-bundle-newrelic-infrastructure-license
+        - name: NRIA_VERBOSE
+          value: "1"
+        ```
 
-      ```shell
-      kubectl apply -f your_newrelic_k8s.yaml -n <NEWRELIC_NAMESPACE>
-      ```
+       Once you have edited the manifest, upgrade the solution with the following command:
```
[the remark this answers is not in the document]
Comment (nbaenam) [reply src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/get-logs-version.mdx:67]: Modified!

File: src/content/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/find-use-your-kubernetes-data.mdx [hunk 6]
```diff
@@ -331,108 +334,14 @@ To learn more about these events and their attributes, see [our data dictionary]
 * `deploymentName`
 * `namespaceName`
 
-## Manage alerts [#alerts]
```
[the remark this answers is not in the document]
Comment (nbaenam) [reply src/content/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/find-use-your-kubernetes-data.mdx:334]: Yes, we have a whole section on how to manage alerts on Kubernetes. That content is duplicated. Thanks for the comment, we never know!

File: src/content/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-events-integration.mdx [hunk 7]
```diff
@@ -1,93 +1,126 @@
...
+    - Integrations
+    - Kubernetes integration
+    - Kubernetes events
 metaDescription: The Kubernetes events integration for infrastructure monitoring watches for events happening in Kubernetes clusters and sends them to New Relic.
 redirects:
-  - /docs/integrations/kubernetes-integration/kubernetes-events/install-kubernetes-events-integration
-  - /docs/new-relic-kubernetes-events-integration
-  - /docs/integrations/kubernetes-integration/kubernetes-events/new-relic-kubernetes-events-integration
-  - /docs/kubernetes-pixie/kubernetes-integration/kubernetes-events/install-kubernetes-events-integration
+    - /docs/integrations/kubernetes-integration/kubernetes-events/install-kubernetes-events-integration
```
[the remark this answers is not in the document]
Comment (nbaenam) [reply src/content/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-events-integration.mdx:9]: I didn't know about this convention for front matter... I usually hit a tab key and check that it is not red. I'll ask our eng team for the correct way to do this ;-)

File: src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx [hunk 8]
```diff
@@ -1,121 +1,128 @@
...
+    - Enable log monitoring in New Relic
 metaDescription: 'Install and configure New Relic logging for Kubernetes, so you can use enhanced log management capabilities.'
 redirects:
-  - /docs/enable-new-relic-logs-kubernetes
-  - /docs/logs/new-relic-logs/enable-logs/enable-new-relic-logs-kubernetes
-  - /docs/logs/enable-logs/enable-logs/kubernetes-plugin-logs
-  - /docs/logs/enable-log-monitoring-new-relic/enable-log-monitoring-new-relic/kubernetes-plugin-log-forwarding
-  - /docs/integrations/kubernetes-integration/logs/get-kubernetes-log-data
-  - /docs/logs/enable-log-management-new-relic/enable-log-monitoring-new-relic/kubernetes-plugin-log-forwarding
+    - /docs/enable-new-relic-logs-kubernetes
```
[the remark this answers is not in the document]
Comment (nbaenam) [reply src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx:9]: I've added a message to the help-doc-eng channel. I've seen several variations throughout the site...

File: src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx (hunk 1 above)
Comment (nbaenam) [reply src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx:56]: @mangulonr, what do you think?

File: src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx (hunk 2 above)
Comment (nbaenam) [reply src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx:68]: Modified!

**Pull request merged**